### PR TITLE
docs(aio): Updated Amexio resource description

### PR DESCRIPTION
**Pull request by ketan-gote**
docs(aio): Updated Amexio resource description

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ X] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ X] No

### CODE REVIEW

**Comment by ketan-gote** [comment]
@jdalton any update on this PR

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Thanks for this, @ketan-gote, I've just small suggestions relating to typos and punctuation.

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -414,7 +414,7 @@
             "url": "https://www.jqwidgets.com/angular/"
           },
           "amexio": {
-            "desc": "Amexio (Angular MetaMagic EXtensions for Inputs and Outputs)  is a rich set of Angular components powered by Bootstrap for Responsive Design. UI Components include Standard Form Components, Data Grids, Tree Grids, Tabs etc. Open Source (Apache 2 License) & Free and backed by MetaMagic Global Inc",
+            "desc": "Amexio is a rich set of Angular components powered by HTML5 & CSS3 for Responsive Web Design and 80+ built-in Material Design Themes. Amexio has 3 Editions, Standard, Enterprise and Creative. Std Edition consists of basic UI Components which include Grid, Tabs, Form Inputs etc, while Enterprise Edition consists of components like Calendar, Tree Tabs, Social Media Logins (Facebook, GitHub, Twitter etc) and Creative Edtion is focused building elegant and beautiful websites. With more than 200+ components/features, all the editions are open-sourced and free based on Apache 2 License.",
```
[the remark this answers is not in the document]
Comment (kapunahelewong) [reply aio/content/marketing/resources.json]: Strange, the diff isn't showing. I'll pull out the parts I added:

1. "etc" -> etc.
>...include Grid, Tabs, Form Inputs etc., while Enterprise Edition consists of components like Calendar, Tree Tabs, 

and here:
>...Social Media Logins (Facebook, GitHub, Twitter etc.) 

2. Creative Edtion  --> Edition

3. "free based" -> free, based
> open-sourced and free, based on Apache 2 License.",

**Comment by kapunahelewong** [comment]
Oh, and and could you squash and rebase? You only want one commit to be merged. I have some [notes on my workflow](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase) if you'd find them helpful.

**Comment by jbogarthyde** [comment]
@ketan-gote Can you please amend the commit message? It can just be "docs: updated amexio resource description" -- the (aio) part is for doc infrastructure changes.

Also, another small nit on the content. Our author guidelines call for avoiding latin abbreviations altogether -- you can replace "etc." with "**and so on**". 
(Similarly, we prefer "**that is**" to "i.e." and "**for example**" to "e.g.")

**Comment by ketan-gote** [comment]
Closing this pull request, will open other.

**Pull request closed without merging**